cpp/velox/compute/VeloxRuntime.cc

```diff
@@ -65,6 +70,14 @@ VeloxRuntime::VeloxRuntime(
   debugModeEnabled_ = veloxCfg_->get<bool>(kDebugModeEnabled, false);
   FLAGS_minloglevel = veloxCfg_->get<uint32_t>(kGlogSeverityLevel, FLAGS_minloglevel);
   FLAGS_v = veloxCfg_->get<uint32_t>(kGlogVerboseLevel, FLAGS_v);
+  FLAGS_velox_exception_user_stacktrace_enabled =
+      veloxCfg_->get<bool>(kEnableUserExceptionStacktrace, FLAGS_velox_exception_user_stacktrace_enabled);
```

Comment:
In VeloxBackend.cc, one variable holding the default value is used as it shows below. Better to use the same default variable for consistency? Though they maybe hold the same bool value currently.
```
backendConf_->get<bool>(kEnableUserExceptionStacktrace, kEnableUserExceptionStacktraceDefault);
```

Ditto for the following config if applicable.

Reply:
It is the same actually. `SQLConf.get.getALL` would get all user-configured confs and passed to here, so the default value here is not necessary.